SA database "TestTimeouts" unit test flaky

```
[mysql] 2017/02/15 01:49:06 packets.go:33: read tcp 172.17.0.5:45500->172.17.0.3:3306: i/o timeout
[mysql] 2017/02/15 01:49:06 packets.go:33: read tcp 172.17.0.5:45502->172.17.0.3:3306: i/o timeout
[mysql] 2017/02/15 01:49:06 packets.go:33: read tcp 172.17.0.5:45504->172.17.0.3:3306: i/o timeout
--- FAIL: TestTimeouts (0.32s)
	database_test.go:94: Got wrong type of error: driver: bad connection
FAIL
coverage: 70.8% of statements
FAIL	github.com/letsencrypt/boulder/sa	1.863s
```

I saw this build failure on #2563 this morning (A README only change), and on a merge to master yesterday from a PR that was green from CI.

Seems flaky, but we should diagnose fully.

--- a/sa/database_test.go
+++ b/sa/database_test.go
@@ -75,14 +75,14 @@ func TestStrictness(t *testing.T) {
 }
 
 func TestTimeouts(t *testing.T) {
-	dbMap, err := NewDbMap(vars.DBConnSA+"?readTimeout=100ms", 1)
+	dbMap, err := NewDbMap(vars.DBConnSA+"?readTimeout=1s", 1)
 	if err != nil {
 		t.Fatal("Error setting up DB:", err)
 	}
 	// SLEEP is defined to return 1 if it was interrupted, but we want to actually
 	// get an error to simulate what would happen with a slow query. So we wrap
 	// the SLEEP in a subselect.
-	_, err = dbMap.Exec(`SELECT 1 FROM (SELECT SLEEP(1)) as subselect;`)
+	_, err = dbMap.Exec(`SELECT 1 FROM (SELECT SLEEP(5)) as subselect;`)
 	if err == nil {
 		t.Fatal("Expected error when running slow query, got none.")
 	}

